README.md

```diff
@@ -32,6 +32,7 @@ $ ./3d-ascii-viewer --help
 
 * [Fox and ShibaInu models](https://opengameart.org/content/fox-and-shiba) made by PixelMannen for the Public Domain (CC0).
 * [Tree models](https://opengameart.org/content/fox-trees-pack) made by Lokesh Mehra (mehrasaur) for the Public Domain (CC0).
+* [PLY models](https://people.sc.fsu.edu/~jburkardt/data/ply/ply.html) made by John Burkardt under the LGPL license.
```

Comment:
The LGPL license is less permissive than the MIT license. So, I think that it is not correct to distribute LGPL-protected content with this MIT licensed repository. Referring to it is fine though.

I am no expert in legal stuff, but my **mediocre** understanding is that the LGPL, like the GPL, protects with copyleft the content, but is a little more relaxed in the sense that it allows the content to be distributed in for-profit applications as long as some guarantees for the user are protected. The MIT license, on the other hand, allows the software to be used for any purpose, even commercial. 

I choose the MIT license since this is a fairly small project and I would like it to be useful for as many people as possible, so I don't think we should change the license. Also, there is a contributor interested in FreeBSD portability (the BSD license is quite similar to the MIT license, also very permissive). See #1 .

Reply:
I removed the models from the commits for now. It would be nice if we could find some CC0 (public domain) PLY models, or export some CC0 models as PLY and include them in the repo.